File: 백준/Bronze/14724. 관리자는 누구？/관리자는 누구？.c
#include <stdio.h>

int main() {
    char* names[] = {"PROBRAIN", "GROW", "ARGOS", "ADMIN", "ANT", "MOTION", "SPG", "COMON", "ALMIGHTY"};
    int N;
    scanf("%d", &N);
    
    int idx = 0, candidate = 0;
    for (int i = 0; i < 9; i++) {
        int tmp = 0;
        for (int j = 0; j < N; j++) {
            int score;
            scanf("%d", &score);
            if (score > tmp) {
                tmp = score;
            }
        }
        if (candidate < tmp) {
            candidate = tmp;
            idx = i;
        }
    }
    printf("%s\n", names[idx]);
    return 0;
}

File: 백준/Bronze/14724. 관리자는 누구？/관리자는 누구？.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String[] names = {"PROBRAIN", "GROW", "ARGOS", "ADMIN", "ANT", "MOTION", "SPG", "COMON", "ALMIGHTY"};
        
        int N = sc.nextInt();
        sc.nextLine();
        
        int idx = 0, candidate = 0;
        for (int i = 0; i < 9; i++) {
            String[] inputLine = sc.nextLine().split(" ");
            int tmp = 0;
            for (String numStr : inputLine) {
                int c = Integer.parseInt(numStr);
                if (c > tmp) {
                    tmp = c;
                }
            }
            if (candidate < tmp) {
                candidate = tmp;
                idx = i;
            }
        }
        System.out.println(names[idx]);
        sc.close();
    }
}